packages/@react-aria/interactions/src/useHover.ts

```diff
@@ -16,30 +16,34 @@
 // See https://github.com/facebook/react/tree/cc7c1aece46a6b69b41958d731e0fd27c94bfc6c/packages/react-interactions
 
 import {HoverEvents} from '@react-types/shared';
-import {HTMLAttributes, useMemo, useRef} from 'react';
+import {HTMLAttributes, useMemo, useRef, useState} from 'react';
 
 export interface HoverProps extends HoverEvents {
   /** Whether the hover events should be disabled. */
   isDisabled?: boolean
+  isHovered?: boolean
```

Comment:
Would you mind adding some tests to cover isHovered? https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/interactions/test/useHover.test.js

Reply:
Sure! 
I added them, could you check if they are covered properly?
I checked other tests like `usePress` but couldn't find similar tests 😅